extensions/type-the-alphabet/src/typethealphabet.jsx

```diff
@@ -23,7 +32,7 @@ function Statistics({ letterTimes, totalTime, onReset, pop }) {
             }}
             icon={Icon.RotateClockwise}
           />
-          <Action.CopyToClipboard title="Copy Statistics" content={markdown} />
+          <CopyToClipboardAction title="Copy Statistics" content={markdown} />
```

Comment:
```suggestion
          <Action.CopyToClipboard title="Copy Statistics" content={markdown} />
```
Why was this changed?